index.js

```diff
@@ -458,9 +458,13 @@ class SortableListView extends React.Component {
   }
 
   componentDidMount() {
-    InteractionManager.runAfterInteractions(() => {
-      this.timer = setTimeout(() => this && this.measureWrapper(), 0)
-    })
+    if (this.props.runAfterInteractions) {
```

Comment:
`runAfterInteractions` is not documented, now I think a prop like that is necessary as seems a lot people are facing the problem, we should provide a switch to turn on/off that
Coulud you add document for the new prop? BTW I think `measureAfterInteractions` would be better?

Reply:
@nihgwu, the idea was  to replace the function with,

```javascript
componentDidMount() {
this.timer = setTimeout(() => this && this.measureWrapper(), 5)
}
```
if we keep `InteractionManager.runAfterInteractions(() => {`, it will give us an error,